packages/nhsuk-frontend-review/src/index.njk

```diff
@@ -100,6 +100,12 @@
           }
         ]
       }) }}
+
+      <hr class="nhsuk-section-break nhsuk-section-break--xl nhsuk-section-break--visible">
+
+      <h2 class="nhsuk-heading-l">Documentation</h2>
+      <p class="nhsuk-body"><a class="nhsuk-link" href="docs/sassdoc/">Sass API reference</a></p>
+
```

Comment:
The components section is so long I worry about this getting missed. Would it be crazy to put at the top?

Perhaps an alternate (future pr?) could be basic nav / index page to link between the three sorts of things.